components/atom/button/src/index.js

```diff
@@ -0,0 +1,92 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const CLASS = 'sui-AtomButton'
+const TYPES = ['primary', 'accent', 'secondary', 'tertiary']
+const MODIFIERS = ['disabled', 'small', 'large', 'fullWidth', 'focused']
+
+const appendClass = (suffix) => `${CLASS}--${suffix}`
+const includes = (array, item) => array.indexOf(item) !== -1
+const getTypes = (props) => {
```

Comment:
If I'm not wrong, doing it this way, last type wins. Although the developer should avoid having more than one type set as prop, consider that it might be dangerous as maybe some type in between applies styling that is not overwritten afterwards.

Reply:
Changed it so we'll only have one type no matter how many are defined.